nncf/common/accuracy_aware_training/runner.py

```diff
@@ -161,28 +176,29 @@ def create_training_loop(self) -> TrainingRunner:
         if nncf_backend is BackendType.TORCH:
             from nncf.torch.accuracy_aware_training.runner import PTAccuracyAwareTrainingRunner
             return PTAccuracyAwareTrainingRunner(self.accuracy_aware_training_params, self.lr_updates_needed,
-                                                 self.verbose, self.validate_every_n_epochs,
-                                                 self.dump_checkpoints)
+                                                 self.verbose, self.dump_checkpoints)
         if nncf_backend == BackendType.TENSORFLOW:
             from nncf.tensorflow.accuracy_aware_training.runner import TFAccuracyAwareTrainingRunner
             return TFAccuracyAwareTrainingRunner(self.accuracy_aware_training_params,
-                                                 self.verbose, self.validate_every_n_epochs,
-                                                 self.dump_checkpoints)
+                                                 self.verbose, self.dump_checkpoints)
         raise RuntimeError('Got an unsupported value of nncf_backend')
 
 
 class AdaptiveCompressionLevelTrainingRunnerCreator(TrainingRunnerCreator):
-    def __init__(self, accuracy_aware_training_params, compression_controller,
-                 lr_updates_needed, verbose, minimal_compression_rate,
-                 maximal_compression_rate, validate_every_n_epochs,
```

Comment:
Why did you remove `validate_every_n_epochs`?

Reply:
I placed it in `accuracy_aware_training_params` as other parameters from the config file

Reply:
Are you satisfied?

Reply:
Ok, get it.